xbmc/Application.cpp

```diff
@@ -211,7 +211,7 @@
 #if defined(TARGET_ANDROID)
 #include "platform/android/activity/XBMCApp.h"
 #include "platform/android/activity/AndroidFeatures.h"
-#include "platform/android/jni/Build.h"
+#include "androidjni/Build.h"
```

Comment:
Not sure, more a coding convention, really, imho.
My approach would be that brackets should be reserved to base system includes, but I have no strong opinion.
I'd follow @Montellese opinion on this